app/src/app/api/v0/city/[city]/boundary/route.ts

```diff
@@ -1,35 +1,17 @@
 import { GLOBAL_API_URL } from "@/services/api";
 import { logger } from "@/services/logger";
 import { apiHandler } from "@/util/api";
-import createHttpError from "http-errors";
 import { NextResponse } from "next/server";
-import wellknown from "wellknown";
+import CityBoundaryService from "@/backend/CityBoundaryService";
 
 export const GET = apiHandler(async (_req, { params }) => {
   const url = `${GLOBAL_API_URL}/api/v0/cityboundary/city/${params.city}`;
   logger.info(`Fetching ${url}`);
 
   try {
-    const boundary = await fetch(url);
+    const boundaryData = await CityBoundaryService.getCityBoundary(params.city);
 
-    const data = await boundary.json();
-
-    if (!data.city_geometry) {
-      throw new createHttpError.NotFound(
-        `City boundary for locode ${params.city} not found`,
-      );
-    }
-
-    const wtkData = data.city_geometry;
-    const geoJson = wellknown.parse(wtkData);
-    const boundingBox = [
-      data.bbox_west,
-      data.bbox_south,
-      data.bbox_east,
-      data.bbox_north,
-    ];
-
-    return NextResponse.json({ data: geoJson, boundingBox, area: data.area });
+    return NextResponse.json({ ...boundaryData });
   } catch (error: any) {
     logger.error(error);
     return NextResponse.json({ error: error.message });
```

Comment:
Please throw an error created by `createHtttpError` instead, otherwise this response will not have the right error code and will go unnoticed when looking for issues.

Otherwise you can also remove the try/ catch statement and let the API handle the error, which will produce the right error code (500 probably) and return the error in the standard format.